ALVRClient/Shaders.metal

```diff
@@ -207,6 +207,80 @@ fragment float4 videoFrameFragmentShader_YpCbCrBiPlanar(ColorInOut in [[stage_in
     return float4(color.rgb, 1.0);
 }
 
+fragment float4 videoFrameFragmentShader_rgbPlanar(ColorInOut in [[stage_in]], texture2d<float> in_tex_rgb, constant EncodingUniform & encodingUniform [[ buffer(BufferIndexEncodingUniforms) ]]) {
+// https://developer.apple.com/documentation/arkit/arkit_in_ios/displaying_an_ar_experience_with_metal
+    
+    float2 sampleCoord;
+    if (FFR_ENABLED) {
+        sampleCoord = decompressAxisAlignedCoord(in.texCoord);
+    } else {
+        sampleCoord = in.texCoord;
+    }
+    
+    constexpr sampler colorSampler(mip_filter::linear,
+                                   mag_filter::linear,
+                                   min_filter::linear);
+    
+    float3 rgb_uncorrect = in_tex_rgb.sample(colorSampler, sampleCoord).rgb;
+    
+    const float DIV12 = 1. / 12.92;
```

Comment:
Needs rebasing to use `NonlinearToLinearRGB` and `EncodingNonlinearToLinearRGB` now